src/input/config.rs

```diff
@@ -118,6 +118,9 @@ pub struct NicConfig {
 #[derive(Clone, PartialEq)]
 pub struct FileConfig {
     pub ess: Essential,
+    /// A list of allowed file extensions.
+    /// Each extension should start with a `.` (e.g. `.txt`, `.csv`).
```

Comment:
This should be altered according to the change.

Reply:
I've updated the description to reflect the changes!